src/components/shared/ReconfigurationDrawer.tsx

```diff
@@ -0,0 +1,162 @@
+import { t, Trans } from '@lingui/macro'
+import { Button, Drawer, DrawerProps, Input, Space } from 'antd'
+import { constants, utils } from 'ethers'
+import { NetworkContext } from 'contexts/networkContext'
+import { useContext, useState } from 'react'
+import { ThemeContext } from 'contexts/themeContext'
+import { drawerWidth } from 'utils/drawerWidth'
+
+import ReconfigurationOption from 'components/shared/ReconfigurationOption'
+
+import {
+  ballotStrategies,
+  customStrategy,
+  Strategy,
+} from 'constants/ballotStrategies/ballotStrategies'
+
+function CustomStrategyForm({
+  address,
+  setAddress,
+}: {
+  address?: string
+  setAddress: (address: string) => void
+}) {
+  const { signerNetwork } = useContext(NetworkContext)
+  return (
+    <div>
+      <Input
+        style={{ width: 400 }}
+        value={address}
+        placeholder={constants.AddressZero}
+        onChange={e => setAddress(e.target.value.toLowerCase())}
+      />
+      <p>
+        <Trans>
+          The address of any smart contract deployed on {signerNetwork} that
+          implements
+        </Trans>{' '}
+        <a
+          href="https://github.com/jbx-protocol/juice-contracts-v1/blob/05828d57e3a27580437fc258fe9041b2401fc044/contracts/FundingCycles.sol"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <Trans>this interface</Trans>
+        </a>
+        .
+      </p>
+    </div>
+  )
+}
+
+export default function ReconfigurationDrawer({
```

Comment:
Maybe let's use a more specific name than "ReconfigurationDrawer", like "ReconfigurationStrategyDrawer"? Since we also use "Reconfiguration" to refer to the reconfig form as a whole